pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
     ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = ">=3.4"
```

Comment:
Why is this changed?

Reply:
I want to have more flexibility.  In order to get this working on 3.4 I basically just removed **F-string formatting** and that gave us some backward compatibility.  I think allowing 3.4-3.7 would be optimal.  I changed this in _rinse_ as well.

Using 2.7 would be good too, but I know 2.7 is going out of style next year.  So that's probably not really worth our time.